app/sidekiq/mhv/account_creator_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'mhv/user_account/creator'
+
+module MHV
+  class AccountCreatorJob
+    include Sidekiq::Job
+
+    sidekiq_options retry: false
+
+    def perform(id)
```

Comment:
Do we need an optional param to break the cache when calling the `Creator`?

Reply:
@bramleyjl Actually thinking about it, this job is def a no-op if the call is cached, so it should always break the cache